tools/reproduce_comparison/Jenkinsfile

```diff
@@ -4,7 +4,7 @@ Jenkins job does reproducible build compare.
 import groovy.json.JsonOutput
 
 pipeline {
-    agent any
+    agent { label 'dockerBuild&&linux&&x64' }
```

Comment:
fine with this, but if mac and windows logic are enabled later, we need find a better label combination or even move agent setting into later stage .